website/content/docs/agent/autoauth/methods/azure.mdx

```diff
@@ -17,3 +17,11 @@ on the value of the `resource` parameter.
 - `role` `(string: required)` - The role to authenticate against on Vault
 
 - `resource` `(string: required)` - The resource name to use when getting instance information
+
+- `object_id` `(string: optional)` - The object ID of the user-assigned managed identity to use
+  when acquiring an [access token](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-to-use-vm-token#get-a-token-using-http).
+  Only one of `object_id` or `client_id` may be provided.
+
+- `client_id` `(string: optional)` - The client ID of the user-assigned managed identity to use
+  when acquiring an [access token](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-to-use-vm-token#get-a-token-using-http).
+  Only one of `object_id` or `client_id` may be provided.
```

Comment:
nit: Just to help avoid update/copy-paste mistakes in future.

```suggestion
  when acquiring an [access token][azure-access-token].
  Only one of `object_id` or `client_id` may be provided.

- `client_id` `(string: optional)` - The client ID of the user-assigned managed identity to use
  when acquiring an [access token][azure-access-token].
  Only one of `object_id` or `client_id` may be provided.
  
[azure-access-token]: https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-to-use-vm-token#get-a-token-using-http
```

Reply:
TIL. Thanks, @tomhjp!